pkg/volume/rbd/rbd_util.go

```diff
@@ -124,6 +124,11 @@ func (util *RBDUtil) MakeGlobalPDName(rbd rbd) string {
 	return makePDNameInternal(rbd.plugin.host, rbd.Pool, rbd.Image)
 }
 
+// <><><>
+func (util *RBDUtil) MakeGlobalVDPDName(rbd rbd) string {
+	return makePDNameInternal(rbd.plugin.host, rbd.Pool, rbd.Image)
+}
```

Comment:
MakeGlobalVDPDName should return directory like;
iscsi: `/var/lib/kubelet/plugins/kubernetes.io/iscsi/volumeDevices/iface_name/portal-some_iqn-lun-lun_id`

MakeGlobalPDName returns dir like;
`/var/lib/kubelet/plugins/kubernetes.io/iscsi/iface_name/portal-some_iqn-lun-lun_id`

The difference is path from MakeGlobalVDPDName() contains `volumeDevices`. They are not same.